data/dockerFiles/README.md

```diff
@@ -0,0 +1,15 @@
+# Dockerfiles to create a development environment
+
+In this section you can find some Dockerfiles to build an image that will have all necessary
+dependencies to build cc-oci-runtime source code and run tests.
+
+
+To build and run, execute the next commands:
+
+```bash
+$ cd clearDockerFile
```

Comment:
Yu could put all these `Dockerfile`s into a single directory (`data/dockerFiles/`), and call them for example `Dockerfile.clear`, `Dockerfile.fedora`, `Dockerfile.ubuntu`.

Then, use the `-f` flag to specify which file to use:

```
$ docker build -t clearlinux-cor -f Dockerfile.ubuntu .
```